src/services/k8s/knative.go

```diff
@@ -71,7 +74,63 @@ func loadManifest(project *project.Project, dockerImage docker.DockerImage) (*se
 	return service, nil
 }
 
-func Apply(namespace string, config *rest.Config, project *project.Project, dockerImage docker.DockerImage) error {
+func loadEnvFile(envFile string) ([]corev1.EnvVar, error) {
+	var envVarList []corev1.EnvVar
+	if _, err := os.Stat(envFile); err != nil {
+		if os.IsNotExist(err) {
+			log.Info("No environment variables file to Load!")
```

Comment:
I'm not a particular fan of this behaviour, the application will be deployed with the wrong configuration, potentially breaking the environment.

Similarly having `.env` as default might cause the application to be deployed with values that were only meant to be used in the local environment.

A better approach might be to have a "non guessable" default value like `.env.initium` let's have a chat about this.

Reply:
The default will be loaded from configuration and visible to the user
```bash
>go run main.go init config                                                                            
app-name: initium-cli
cluster-endpoint: null
container-registry: ghcr.io/nearform
default-branch: main
dockerfile-name: null
env-var-file: .env
registry-user: null
runtime-version: null
```

The behaviour imho should be something like:
```
- if file not exist:
   - if file is default: ignore and move one)
   - if file is specified by user: return error
```

Reply:
Few updates were made in order to address the above behaviour.